internal/search/search.go

```diff
@@ -52,3 +52,13 @@ func AtlasClusterExists(clusters []atlas.Cluster, name string) bool {
 
 	return false
 }
+
+// FindPopularRegionIndex returns the index of the default region
+func FindPopularRegionIndex(regions []*atlas.AvailableRegion) int {
```

Comment:
[nit] consider that `search`, the package name, is part of your method name, something like `search.DefaultRegion` could be a better name for this function